samples/sqlservices/src/controllers/mainController.ts

```diff
@@ -53,49 +53,247 @@ export default class MainController implements vscode.Disposable {
 		this.registerSplitPanelModelView();
 		this.registerModelViewDashboardTab();
 
+		vscode.commands.registerCommand('sqlservices.openEditor', () => { this.openEditor(); });
+		vscode.commands.registerCommand('sqlservices.openDialog', () => { this.openDialog(); });
+		vscode.commands.registerCommand('sqlservices.openEditorWithWebView', () => { this.openEditorWithWebview(buttonHtml, counterHtml); });
+		vscode.commands.registerCommand('sqlservices.openEditorWithWebView2', () => { this.openEditorWithWebview2(); });
+		vscode.commands.registerCommand('sqlservices.openWizard', () => { this.openWizard(); });
+		vscode.commands.registerCommand('sqlservices.openModelViewDashboard', () => { dashboard.openModelViewDashboard(this.context); });
+		vscode.commands.registerCommand('sqlservices.updateObjectExplorerNode', async (context: azdata.ObjectExplorerContext) => { await objectExplorer.updateNode(context); });
+
 		azdata.tasks.registerTask('sqlservices.clickTask', (profile) => {
 			vscode.window.showInformationMessage(`Clicked from profile ${profile.serverName}.${profile.databaseName}`);
 		});
 
-		vscode.commands.registerCommand('sqlservices.openDialog', () => {
-			this.openDialog();
-		});
-
 		vscode.commands.registerCommand('sqlservices.openConnectionDialog', async () => {
 			let connection = await azdata.connection.openConnectionDialog();
 			if (connection) {
 				console.info('Connection Opened: ' + connection.options['server']);
 			}
 		});
 
-		vscode.commands.registerCommand('sqlservices.openEditor', () => {
-			this.openEditor();
+		return Promise.resolve(true);
+	}
+
+	//#region Primary entrypoints
+
+	private openDialog(): void {
```

Comment:
I reorganized the code a bit (and renamed some vars) here to make the construction of the tabs and their contents easier to follow.  This is the only change made inside a method; everything else in this diff is moving blocks around so they can be grouped into readable regions.

With everything in place now, the next PR will be readable when I start digging into the individual methods.